.github/workflows/CI.yml

```
@@ -15,17 +15,17 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.0'
-          - '1'
+          - '1.0'  # LTS
+          - '1'    # Latest Release
```

Since we've started using 2x quotes everywhere else

```suggestion
          - "1.0"  # LTS
          - "1"    # Latest Release
```